templates/common/vsphere/files/etc-systemd-system-crio-stream-address.conf.yaml

```diff
@@ -2,20 +2,10 @@ mode: 0644
 path: "/etc/systemd/system/crio.service.d/20-stream-address.conf"
 contents:
   inline: |
-    {{ if .Infra -}}
-    {{ if .Infra.Status -}}
-    {{ if .Infra.Status.PlatformStatus -}}
-    {{ if .Infra.Status.PlatformStatus.VSphere -}}
-    {{ if .Infra.Status.PlatformStatus.VSphere.APIServerInternalIP -}}
```

Comment:
don't we need to replace this with the simpler conditional used in other templates? `{{ if (onPremPlatformAPIServerInternalIP .) -}}`

Reply:
There is no reference to the PlatformStatus in that file. Are you saying you don't want it at all?